Vagrantfile.local.example

```diff
@@ -13,7 +13,7 @@
 # You can configure many vms here if necessary.
 # Adjust memory and CPUs as needed.
 $vms = {
-  "default" => { "fqdn" => "vagrant-test.tag1consulting.dev", "ipaddress" => "10.10.10.10", "memory" => "2048", "cpus" => "2" },
+  "default" => { "fqdn" => "vagrant-test.tag1consulting.dev", "ipaddress" => "10.10.10.10", "memory" => "2048", "cpus" => "2", "forwarded_port" => 80 },
```

Comment:
Unrelated, but while we're changing things I'd rather have this hostname match the standard we've set of <env>-<type>#.domain -- so how about we do vagrant-multi1.tag1consulting.com? And change the hiera host yaml filename and comment to match.